src/api/config/routes.rb

```diff
@@ -328,10 +328,6 @@ def self.public_or_about_path?(request)
       post 'user/register' => :register
       get 'user/register_user' => :register_user
 
-      get 'user/login' => :login
```

Comment:
can we modify the api? or should we keep them at least until the next big release? @adrianschroeter ^^

Reply:
We need to keep it!

Reply:
ok, then I guess we need something like a FIXME to remove them in the next big release. Because we don't want to keep them for ever, right?

Reply:
Because we are looking at the source code all the time. At old issues, not so much.

Reply:
@Ana06 what I don't get is which API you are talking about? This is a WebUI controller. Not an API controller.

Reply:
Hey, @dgdavid @hennevogel is right, those are routes for the webui. You can change those without having to respect any backward compatibility.

So, just change it back to what you did in first place. BTW the route tests you added are nonsense right now, but in general we don't test routes :+1:

Reply:
👍

Reply:
@mdeniz, 

done!